### DISCUSSION TITLE
只有一个账单导入不进去，其他20多个账单都没问题

### DISCUSSION THREAD

**Discussion by asing** [deb-sig/double-entry-generator#61] [Q&A] [answered]
提示信息如下：
`Using config file: ./wechat/config.yaml

[Provider-Wechat] 2022/07/06 17:41:27 Failed to translate: parse error on line 81, column 64: bare " in non-quoted-field
exit status 1`

wechat/config.yaml  81行是我定义的一个消费项
删除掉还是不行...

**Comment by gaocegege** [comment]
你能提供一下 config.yaml 的示例么，可以和谐掉一些敏感内容

**Reply by asing** [reply]
@gaocegege 请问看了么，是什么原因，需要账单文件么？

**Reply by gaocegege** [reply]
如果有账单的话最好，我现在没有复现出来

**Reply by asing** [reply]
能给个邮箱不？我发您

**Comment by Triple-Z** [comment]
请问使用的是哪个版本呢？

```
double-entry-generator version
```

**Reply by asing** [reply]
我的使用方法可能不对，执行后出现

```
double-entry-generator version
zsh: command not found: double-entry-generator
```

我现在的操作命令是
` go run ./ translate --config ./wechat/config.yaml --provider wechat --output ./wechat/**.beancount ./wechat/**.csv`

我之前发过一个帖子没回复，这是我自己研究的

**Reply by Triple-Z** [reply]
按照你的命令，现实版本应该是：

```
go run ./ version
```

**Reply by Triple-Z** [reply]
建议直接在 release 页面下载对应架构二进制版本来使用。

https://github.com/deb-sig/double-entry-generator/releases

**Reply by asing** [reply]
执行是这样的
`2022/07/07 13:29:18 double-entry-generator Version: UNKNOWN, Commit: UNKNOWN, Build Path: UNKNOWN`

建议直接在 release 页面下载对应架构二进制版本来使用。
https://github.com/deb-sig/double-entry-generator/releases
这个版本下载了，但是不知道咋正确使用....

**Reply by Triple-Z** [reply]
解压后在命令行里直接运行就可以了呀。

```
./double-entry-generator version
```

**Reply by Triple-Z** [reply]
> 执行是这样的
> 
> `2022/07/07 13:29:18 double-entry-generator Version: UNKNOWN, Commit: UNKNOWN, Build Path: UNKNOWN`

你这个是自己本地直接通过 Go 来构建的版本，没通过 Makefile 来构建就不会被注入 Git 版本信息。

应该更新下项目代码再运行也是可以的。

**Comment by Triple-Z** [comment, marked as answer]
这个问题似乎在 v1.5.1 就已经解决了。

相关 PR：https://github.com/deb-sig/double-entry-generator/pull/55

**Reply by asing** [reply]
`./double-entry-generator version
2022/07/07 13:35:27 double-entry-generator Version: 1.5.1, Commit: 6486dc0, Build Path: github.com/deb-sig/double-entry-generator`

请教如何使用处理账单....

**Reply by Triple-Z** [reply]
```
./double-entry-generator  translate --config ./wechat/config.yaml --provider wechat --output ./wechat/**.beancount ./wechat/**.csv
```

这样就好了呀，就把你之前使用的 `go run` 换掉就行。

当然配置文件和帐单文件位置你可能需要修改一下。

**Reply by asing** [reply]
可以了，谢谢，学习了，二进制版本就这样使用
能否帮我把这个帖子删除掉，有个config文件发现还是不妥

**Reply by Triple-Z** [reply]
我把存有 config 的那条回复删除了。

问题解决的话可以在某个评论上点一个已解决。
